eng/automation/Invoke-DataPlaneGenerateShippablePackage.ps1

```diff
@@ -0,0 +1,32 @@
+#Requires -Version 7.0
+param (
+  [string]$service,
+  [string]$namespace,
+  [string]$sdkPath,
+  [string]$inputfile,
```

Comment:
There can be multiple input files

Reply:
ok, I will update to support multiple inputfiles